### PR TITLE
Feature/ma 212

### PR DESCRIPTION
**Pull request by viktor-honcharov-gen**

### CODE REVIEW

File: frontend/automation-status/types.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { AUTOMATION_STATUS } from "./automation-status.enum"
+
+export type Automation = {
+  id: string,
+  status: AUTOMATION_STATUS,
+  message?: string,
+}
```
Comment (andriivitrenko) [inline frontend/automation-status/types.ts:7]: add new line

File: env.example.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "REACT_APP_SERVER_URL": "http://localhost:3001"
+}
```
Comment (andriivitrenko) [inline env.example.json:3]: add new line

File: frontend/automations/create-creative-adaptation.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  // load selected records and fields
+  useLoadable(cursor);
+
+  // re-render whenever the list of selected records
+  useWatchable(cursor, ["selectedRecordIds"]);
+  const base = useBase();
+  const view = base.getTableById(cursor.activeTableId).getViewById(cursor.activeViewId);
+
+  const [data, setData] = useState<object>(initialData);
+  const [errors, setErrors] = useState<object[]>([]);
```
Comment (andriivitrenko) [inline frontend/automations/create-creative-adaptation.tsx:50]: create interfaces for state

File: frontend/custom-ui/block-control.tsx [hunk 4]
```diff
@@ -60,6 +59,16 @@ function BlockControl({ data, handleChange, path, schema }: BlockControlProps) {
...
+      )
+      .filter(
+        (choice) =>
+          !Boolean(
+            blocks
+              .map((block) =>
+                [block.name, block.attribute, block.category].join("_")
+              )
+              .includes([block.name, block.attribute, choice.name].join("_"))
+          )
```
Comment (andriivitrenko) [inline frontend/custom-ui/block-control.tsx:71]: looks like you only compare `choice.name` and `block.category`

**Review by andriivitrenko — CHANGES_REQUESTED** [review]

**Review by olesmarola — APPROVED** [review]

**Pull request merged**